packages/parmap/parmap.0.9.1/opam

```diff
@@ -1,11 +1,7 @@
-opam-version: "1"
-maintainer: "contact@ocamlpro.com"
+opam-version: "0.9.1"
```

Comment:
`opam-version` should be the version of `opam` needed to read the files. Nowadays "1.2" is a good value.